force-app/main/default/classes/ALLO_Allocations_TDTM.cls

```diff
@@ -1176,4 +1194,56 @@ public class ALLO_Allocations_TDTM extends TDTM_Runnable {
         return (val != null) ? val : defaultVal;
     }
 
+    /**
+     * @description Recreates the Scheduled Payment Allocations for Opportunities whose Allocations were updated as a
+     *              result of A Recurring Donations GAU Allocations being updated.
+     */
+    @testVisible private void recreateScheduledPaymentAllocationsIfNecessary() {
+        if (!isPaymentAllocationsEnabled() || updatedRecurringDonationOppIds.isEmpty()) {
+            return;
+        }
+
+        DmlWrapper paDMLWrapper =  new DMLWrapper();
+
+        // Payment with Payment Allocations Query for Scheduled Payments related to Opportunities
+        // whose Allocations were recreated because the Recurring Donations Allocations changed
+        String pmtReviewQuery = 'SELECT     Id, ' +
+                                ' (SELECT Id FROM Allocations__r) ' +
```

Comment:
Will this `Allocations__r` reference be an issue in managed orgs?

Reply:
I do not believe so, since it is in the same namespace.  If you want, I can namespace qualify it conditionally, but I do not believe it is needed.

Reply:
The 2GP feature test passed, as well.

Reply:
Heh, I thought about that after I posted this, should be fine!